src/explore-education-statistics-frontend/src/modules/find-statistics/components/PublicationList.tsx

```diff
@@ -11,7 +11,12 @@ function PublicationList({ publications }: Props) {
     <ul className="govuk-!-margin-top-0">
       {publications.map(({ id, legacyPublicationUrl, slug, title }) => (
         <li key={id} className="govuk-!-margin-bottom-0">
-          <h3 className="govuk-heading-s govuk-!-margin-bottom-0">{title}</h3>
+          <h3
+            id="publication-heading"
```

Comment:
Duplicate ids will appear in the DOM if this is in a loop - you'd need to include the `id` in there to make it unique.

Reply:
I forgot that the convention for id attributes is that they should be unique. I've switched to using `data-testid` in this case.

Reply:
I think you'd find the same issue with data-testid - worth a Google, but I *think* they're supposed to be unique too.